src/GeositeFramework/js/lib/jquery-ui-slider-pips.js

```diff
@@ -1,583 +1,566 @@
 /*! jQuery-ui-Slider-Pips - v1.11.4 - 2016-10-02
 * Copyright (c) 2016 Simon Goellner <simey.me@gmail.com>; Licensed MIT */
 
-(function( factory ) {
+var pips = {};
```

Comment:
Apologies for this diff; the high level summary of this is that:

1. it drops the IIFE factory function thing which pips would otherwise use to attach itself to `$.slider`
2. it exports the internal function it uses to extend $.slider as a callable function, which will then get called in `BasePlugin`'s constructor

Nothing internal to the function changed though, and almost all of the diff here just updates the function's lines to have the proper indentation.